src/SFA.DAS.EmployerIncentives/Models/EmployerIncentives/ApprenticeshipEmployerType.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel;
+
+namespace SFA.DAS.EmployerIncentives.Models.EmployerIncentives
+{
+    public enum ApprenticeshipEmployerType : byte
```

Comment:
Is this the same as the one that is in `SFA.DAS.EmployerIncentives.Models.Commitments` or should be the same?

Reply:
No, it's different in both inner apis